src/components/HomePage.vue

```diff
@@ -0,0 +1,35 @@
+<script setup>
+  import { ref } from 'vue';
+  import ToDoList from './ToDoList.vue';
+  import HeaderComponent from './HeaderComponent.vue';
+  import toDoListMT from './../assets/toDoListMT.svg';
```

Comment:
Can you give a more explicit name to this image? It is not very clear what this image is from the name of it, something like todoListPlaceholder